File: Kattis/Tarifa/tarifa.cpp
#include <iostream>
using namespace std;
int main() {
  int gb; cin >> gb;
  if (gb >= 1 && gb <= 100) {
    int month;  cin >> month;
    if (month >= 1 && month <= 100) {
      int tot = gb;
      for (int i = 0;i < month;i++) {
        int p;  cin >> p;
        tot = (gb+tot)-p;
      }
      cout << tot;
    }
  }
}

File: Tarifa/tarifa.rb
gb = gets.to_i;
if gb >= 1 and gb <= 100
	month = gets.to_i;
	if month >= 1 and month <= 100
		tot = 0;
		month.times do
			p = gets.to_i;
			tot = (gb+tot)-p;
		end
		tot += gb;
		print tot;
	end
end